src/common/error/internalErrorCode.ts

```diff
@@ -44,6 +44,7 @@ export enum InternalErrorCode {
         RNTempFolderDeletionFailed = 707,
         DebuggingFailedInNodeWrapper = 708,
         PlatformNotSupported = 708,
```

Comment:
That is definitely a bug, good spotting. You should probably bump it up to differentiate them.